README.md

```diff
@@ -1,30 +1,45 @@
-# Virtual list
+# &lt;virtual-list&gt;
 
-Implementation of a Virtual list API, influenced by the research done by the [infinite list study group](https://github.com/domenic/infinite-list-study-group).
+`<virtual-list>` renders only the visible portion of a list and optimizes the amount of DOM operations and reflows.
 
-## Local Setup
+## Properties
 
-Ensure you have installed the npm dependencies and serve from the project root
-```sh
-$ npm install
-$ polymer serve --npm
-```
-Then, navigate to the url: http://localhost:8081/components/virtual-list/
+- _newChild_
+  - type: `function(item: any, index: number) => Element`
+  - returns an `Element` for the given item and index.
+- _updateChild_
+  - type: `function(child: Element, item: any, index: number)`
+  - invoked after the `child` is appended to the DOM and requires to be updated with data.
+- _recycleChild_
+  - type: `function(child: Element, item: any, index: number)`
+  - invoked before the `child` is removed from the DOM. Set to control DOM removal/recycling behavior.
+- _items_
+  - type: `Array`
+  - the data model.
+- _layout_ 
+  - type: `string`
+  - layout type, can be set via attribute or property to "vertical" (default) or "horizontal".
 
-## Concepts and Domain
 
-An infinite list optimizes the rendering of DOM according to the visible area and available data.
+## Methods
 
-Some implementations append DOM incrementally, others recycle the DOM.
+- _requestReset()_
+  - Schedules a rerendering of the displayed items.

```

Comment:
We should file a follow-up issue on figuring out the best name for this (assuming we want to keep it).

Reply:
tracking issue: https://github.com/PolymerLabs/virtual-list/issues/26